src/instruction.rs

```diff
@@ -511,34 +511,410 @@ impl Instruction {
                 cycles: 5
             },
 
-
-
-
+            //0xb0 through 0xff - DS
+            0xb0 => Instruction {
+                op: Operation::ORA(Register::B),
```

Comment:
You will need to add all of the new operations you have added here to the Operation enum. This should not compile otherwise.